src/Standards/MySource/Sniffs/Channels/DisallowSelfActionsSniff.php

```diff
@@ -108,7 +108,9 @@ public function process(File $phpcsFile, $stackPtr)
                 // Function was not in this class, might have come from the parent.
                 // Either way, we can't really check this.
                 continue;
-            } else if ($foundFunctions[$funcData['name']] === 'public') {
+            }
+
+            if ($foundFunctions[$funcData['name']] === 'public') {
```

Comment:
You could even choose to add this condition (reversed) to the previous `if` and lower the indent level for the code.